scenarios/centos-9/ceph_backends.yml

```diff
@@ -26,11 +26,20 @@ pre_deploy:
     type: playbook
     source: kustomize_cr.yml
 
+cifmw_services_manila_enabled: true
+
+pre_tests:
+  - name: Create manila resources
+    type: playbook
+    source: manila_create_default_resources.yml
+
 cifmw_run_tests: true
 cifmw_tempest_default_groups: &test_operator_list
   - openstack-operator
   - cinder-operator
   - keystone-operator
+  - manila-operator
```

Comment:
In the Prow job, we are skipping share groups tests, others are automatically skipped. So I am expecting to have the same amount of tests running and passing. If share group tests are running, is because tempest profile.yaml is not working. Trying to fix that.